serverless/networking/serverless-domain-mapping.adoc

```diff
@@ -0,0 +1,13 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-domain-mapping"]
+= Mapping a custom domain name to a service
+:context: serverless-domain-mapping
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+Knative Services are automatically assigned a default domain name based on your cluster configuration. For example, `mysvc.mynamespace.mydomain`.
```

Comment:
That's just a typo, it should be services.
RE the domain names, I think the options there (e.g. `node-1`) will be confusing for users, because we don't mention nodes anywhere in our docs and just refer to services. Would something like `<service_name>.<namespace>.example.com` be acceptable?

Reply:
@abrennan89, sure, the key is just `example.com` because it's always safe. We probably need to expand the guidance to apply to any use of an example domain name; this was directed at just nodes I guess.